tests/src/core/Polyfill.test.ts

```
@@ -35,4 +35,39 @@
+    };
+    
+    window.AudioContext = (window as any).webkitAudioContext;
+    
+    expect(window.AudioContext).to.equal((window as any).webkitAudioContext);
+      
+    const context = new window.AudioContext();
+    
+    const originalDecodeAudioData = context.decodeAudioData;
+    
+    AudioContext.prototype.decodeAudioData = function(arrayBuffer: ArrayBuffer): Promise<AudioBuffer> {
+      const self = this;
```

Biome (1.9.4)

[error] This aliasing of this is unnecessary.

Arrow functions inherits `this` from their enclosing scope.
Safe fix: Use this instead of an alias.


(lint/complexity/noUselessThisAlias)